### PR TITLE
[ET-876] Fix double RSVP capacity problem

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ET-876]

### 📹 🗒️ Screencast: http://p.tri.be/XenSd9

**Before Fix**
Screen Shot 2020-10-26 at 1 21 44 PM

**After Fix**
Screen Shot 2020-10-26 at 1 20 30 PM

[ET-876]: https://moderntribe.atlassian.net/browse/ET-876

### CODE REVIEW

File: src/template-tags/tickets.php [hunk 1]
```diff
@@ -1085,7 +1085,13 @@ function tribe_get_event_capacity( $post ) {
 
 		$provider = Tribe__Tickets__Tickets::get_event_ticket_provider_object( $post_id );
 
-		if ( empty( $provider ) ) {
+		$has_provider = ! empty( $provider );
+
+		if ( ( ! $has_provider && $rsvp_tickets ) || is_a( $provider, 'Tribe_Tickets_RSVP' ) ) {
+			// If we have no provider but have RSVP tickets, or the provider is RSVP, return the RSVP capacity.
+			return (int) $rsvp_cap;
+		} elseif ( ! $has_provider ) {
```
Comment (tr1b0t) [inline src/template-tags/tickets.php:1093]: :no_entry_sign: is_a()? Really? Use instanceof instead.

<sub>hash: 6b79c7123cb12afe92b0af3a73cc9397</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

**Comment by sc0ttkclark** [comment]
Needs change log

**Review by bordoni — CHANGES_REQUESTED** [review]
Let's get a changelog here.

**Review by bordoni — APPROVED** [review]

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**